conda-requirements.txt

```diff
@@ -15,4 +15,5 @@ zstandard
 attrs
 click
 prompt-toolkit
-pyyaml
\ No newline at end of file
+pyyaml
+sphinx-click
```

Comment:
For both pip and conda, this should be part of the test requirements, not the runtime / prod requirements.

Reply:
done.